rss-importer.php

```diff
@@ -128,6 +131,11 @@ function get_posts() {
 				$post_content = $wpdb->escape(html_entity_decode(trim($item->description)));
 			}
 
+			// M.G 09-Aug-2019, if there is an image append it to the content 			
+			if (isset($item->enclosure)) {				
+        		    $post_content = $post_content .' <img src='. $item->enclosure['url'] .' /> ';
```

Comment:
Enclosure isn't necessarily an image. In fact, the most common case I've seen it used in is podcasts referencing mp3 files, for which this code would produce somewhat invalid output.

You could check for the enclosure's type attribute and only do this for ones that start with `image/`. More elaborately, you could also generate `audio` or `video` tags for enclosures of those types.

Reply:
Hi, yes I realize this hence my comment that this code is good just for image.
You might want to add media import to your rss importer in the way you see fit.
Regards,